### PR TITLE
use addrtype match to return the LOCAL trafik

### PR DESCRIPTION
**Pull request by dennybaa**

### CODE REVIEW

**Comment by brianmay** [comment]
Looks like the tests might need updating...

**Comment by dennybaa** [comment]
obviously, though I have no understanding of them and unfortunately have no time to contribute the tests too.

**Comment by dennybaa** [comment]
@brianmay Anyways I'd like to say thank you, sshuttle does it's job and is pretty amazing. I was only missing this small thing since my primary scenario is linux and docker, so I don't pretty much care about anything except nat&iptables. Sorry I have no time to look into the tests.

**Comment by brianmay** [comment]
Merged as part of #337

File: sshuttle/methods/nat.py [hunk 1]
```diff
@@ -50,6 +50,12 @@ def _ipm(*args):
         _ipt('-I', 'OUTPUT', '1', *args)
         _ipt('-I', 'PREROUTING', '1', *args)
 
+        # Firstly we always skip all LOCAL addtrype address, i.e. avoid
+        # tunnelling the traffic designated to all local TCP/IP addresses.
+        _ipt('-A', chain, '-j', 'RETURN',
+                '-m', 'addrtype',
+                '--dst-type', 'LOCAL')
```
Comment (chemikadze) [inline sshuttle/methods/nat.py]: It looks like this is breaking use of local DNS servers. Previos versions were working by matching only TCP traffic. Is matching of non-TCP intentional here?

**Pull request closed without merging**